packages/viewer/src/components/scene-tree-table-cell/scene-tree-table-cell.tsx

```diff
@@ -324,4 +339,18 @@ export class SceneTreeTableCell {
   ): void => {
     tree.toggleExpandItem(node);
   };
+
+  private clearLongPressTimer(): void {
```

Comment:
Should we invoke the `clearLongPressTimer()` function on the disconnected callback?

Reply:
Good idea! Added.